packages/jest-circus/src/eventHandler.ts

```diff
@@ -148,7 +148,11 @@ const eventHandler: Circus.EventHandler = (event, state): void => {
       break;
     }
     case 'test_retry': {
+      // Clear errors so tests can be retried (and not immediately fail)
       event.test.errors = [];
+      // Clear any snapshot data that occurred in previous test run
+      global.expect.clearState();
```

Comment:
This forces `expect` to be available, while circus is so far not tightly coupled to it (and in theory built to be used anywhere, the `legacy-code` directory aside). Can we make it gracefully handle `expect`/`clearState` not being available and move it to a separate event handler?

Reply:
This is a great point, you are right we should find to prevent `jest-circus` of being coupled with `expect`.